package.json

```diff
@@ -38,7 +38,9 @@
     "shelljs": "0.5.3",
     "underscore.string": "3.2.2",
     "wordwrap": "1.0.0",
+    "yeoman-assert": "2.1.1",
```

Comment:
I checked and yes, yeoman already has these as dependency, i think its better to let it be manged by Yeoman than we define it, it might cause conflicts. what do you think @pascalgrimaud ?